src/wiki/help-pages/Miscellaneous.md

```diff
@@ -0,0 +1,46 @@
+---
+eleventyNavigation:
+  key: Miscellaneous
+  parent: Help pages
+--- 
+
+## Fix crash with recent Java 8 versions for old forge modpacks
+
+This applies to old forge modpacks that crashe during standup with the new version of java.
+The crash may look as follows:
+
+```text
+2024-02-28 10:35:15 [SEVERE] [Forestry] myrathi.flatsigns.FlatSigns failed validation. Halting runtime for security reasons. Please replace your mods with untampered versions from the official download sites.
+```
+
+```text
+2024-02-28 10:31:56 [SEVERE] [ForgeModLoader] The minecraft jar file:/home/********/.var/app/org.prismlauncher.PrismLauncher/data/PrismLauncher/libraries/com/mojang/minecraft/1.6.4/minecraft-1.6.4-client.jar!/net/minecraft/client/ClientBrandRetriever.class appears to be corrupt! There has been CRITICAL TAMPERING WITH MINECRAFT, it is highly unlikely minecraft will work! STOP NOW, get a clean copy and try again!
+```
+
+```text
+2023-01-23 16:10:09 [SEVERE] [Forestry] ic2.core.IC2 failed validation. Halting runtime for security reasons. Please replace your mods with untampered versions from the official download sites.
+```
+
+### Solution
+
+1. Create a file inside the instance minecraft(`.minecraft`/`minecraft`) folder with the following content:
```

Comment:
If possible, please specify what this file needs to be called, as it is a bit unclear here

Reply:
sure thing, and thank you I forgot about that completely.
Let me know if you find anything that is missing.